securedrop/source_templates/tor2web-warning.html

```diff
@@ -2,6 +2,6 @@
 {% block body %}
 <h1>{{ gettext('Why is there a warning about Tor2Web?') }}</h1>
 <p>{{ gettext('<a href="tor2web.org">Tor2Web</a> is a proxy service that lets you browse Tor Hidden Services (.onion sites) without installing Tor. It was designed to facilitate anonymous publishing.') }}</p>
-<p>{{ gettext('Tor2Web only protects publishers, not readers. If you upload documents to us using Tor2Web, you are <strong>not anonymous</strong> and could be identified by your ISP or the Tor2Web proxy operators. Additionally, since Tor2Web sites typically do not use HTTPS, it is possible that your connection could be MITM\'ed by a capable adversary.') }}</p>
+<p>{{ gettext('Tor2Web only protects publishers (owners of hidden web services), not readers. If you upload documents to us using Tor2Web, you are <strong>not anonymous</strong> and could be identified by your ISP or the Tor2Web proxy operators. Additionally, since Tor2Web sites typically do not use HTTPS, it is possible that your connection could be MITM\'ed by a capable adversary.') }}</p>
```

Comment:
We introduce a term ("publishers"), explain it, then never use it again. I feel like this whole bit should be rewritten to leave out the details about who's protected by hidden services since the enduser might not now what they are. Also, I think the canonical way to name them is now "onion services."

Reply:
@heartsucker do you have a better wording in mind? Not being a native English speaker, I'm a bit hesitant to try that myself ;-)

Reply:
Actually, that seems be covered by the previous paragraph. I think we could actually cut out this whole sentence:

> Tor2Web only protects publishers (owners of hidden web services), not readers.

it's a bit redundant and adds terms that users don't need to actually know in order to understand what's happening.

Reply:
True! I changed that to use exactly the same terms as the sentence above so it is clearer.

Reply:
That kinda addresses it, but I still think we might be over explaining here. Can @ei8fdb give some input here maybe?